tests/components/soundtouch/test_media_player.py

```diff
@@ -365,6 +396,37 @@ async def test_playing_radio(mocked_status, mocked_volume, hass, one_device):
     assert entity_1_state.attributes["media_title"] == "station"
 
 
+async def test_playing_aux(mocked_status, mocked_volume, hass, one_device):
+    """Test playing AUX info."""
+    mocked_status.side_effect = MockStatusPlayingAux
+    await setup_soundtouch(hass, DEVICE_1_CONFIG)
+
+    assert one_device.call_count == 1
+    assert mocked_status.call_count == 2
+    assert mocked_volume.call_count == 2
+
+    entity_1_state = hass.states.get("media_player.soundtouch_1")
+    assert entity_1_state.state == STATE_PLAYING
+    assert entity_1_state.attributes["source"] == "AUX"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)